internal/pkg/scaffold/helm/role.go

```diff
@@ -47,8 +47,6 @@ func CreateRoleScaffold(cfg *rest.Config, chart *chart.Chart) (*scaffold.Role, e
 	roleScaffold := &scaffold.Role{
 		IsClusterScoped:  false,
 		SkipDefaultRules: true,
-		// TODO: enable metrics in helm operator
-		SkipMetricsRules: true,
```

Comment:
Do we still need the `SkipMetricsRules` field?

Reply:
Nope. I'll remove it.